### PR TITLE
Update multi CSI driver related docs

### PR DESCRIPTION
**Pull request by futuretea**
**Related issues**
https://github.com/harvester/harvester/issues/4420

Summarize CSI driver related informations to section `Multi Container Storage Interface support`  in the `StorageClass` page.

I'll create a new PR to harvester/harvesterhci.io to add KB article for Rook Ceph CSI Driver [KB: Using Rook Ceph storage in Harvester](https://github.com/harvester/harvesterhci.io/pull/41)
After Rook Ceph CSI Driver KB and the [KB article for NetApp Trident CSI Driver](https://github.com/harvester/harvesterhci.io/pull/38) merged
I'll create another new PR to add these KB links to the harvester/docs

### CODE REVIEW

File: docs/advanced/settings.md [hunk 1]
```diff
@@ -129,7 +129,7 @@ If you set a username and password for a private registry, the system will autom
 
 _Available as of v1.2.0_
 
-This setting allows you to install a Container Storage Interface (CSI) in your Harvester cluster to support and use external storage as the VM's non-system data partition and leverage different drivers.
+If you install third-party CSI drivers in the Harvester cluster, you need to configure some necessary informations through this setting before using **Backup/Snapshot** related features.
```
Comment (LucasSaintarbor) [inline docs/advanced/settings.md:132]:
```suggestion
If you install third-party CSI drivers in the Harvester cluster, you must configure some necessary information through this setting before using **Backup/Snapshot** related features.
```

File: docs/advanced/storageclass.md [hunk 2]
```diff
@@ -97,21 +97,17 @@ Harvester now supports installing a Container Storage Interface (CSI) in your Ha
 
 :::note
 
-The system partition of the VM can still use Longhorn. Before v1.2.0, Harvester only supports the use of Longhorn to store virtual machine data, and does not support the use of external storage to store VM data.
+The provisoner of the VM image still use Longhorn. Before v1.2.0, Harvester only supports the use of Longhorn to store virtual machine data, and does not support the use of external storage to store VM data.
```
Comment (LucasSaintarbor) [inline docs/advanced/storageclass.md:100]:
```suggestion
The provisioner of the VM image still uses Longhorn. Before v1.2.0, Harvester only supports using Longhorn to store virtual machine data and does not support using external storage to store VM data.
```

File: docs/advanced/storageclass.md [hunk 3]
```diff
@@ -97,21 +97,17 @@ Harvester now supports installing a Container Storage Interface (CSI) in your Ha
...
-### Header Section
-
-- **Name**: Name of the StorageClass
-- **Description** (optional): Description of the StorageClass.
-- **Provisioner**: Select a provisioner.
-
-### Parameters
-
-Edit the [`csi-driver-config`](../advanced/settings.md#csi-driver-config) setting to add the provisioner for the newly added CSI driver.
+After that, users just need to select the desired StorageClass when creating an empty volume or adding a new empty volume to a virtual machine.
```
Comment (LucasSaintarbor) [inline docs/advanced/storageclass.md:110]:
```suggestion
After performing these steps, you must select the desired StorageClass when creating an empty volume or adding a new empty volume to a virtual machine.
```

**Review by LucasSaintarbor — APPROVED** [review]
Small NITs. LGTM

**Review by guangbochen — APPROVED** [review]

**Pull request merged**